### ISSUE TITLE
CppWinRTStaticProjectReferences doesn't respect ReferenceOutputAssembly

### ISSUE THREAD

**Issue by DHowett-MSFT** [microsoft/xlang#519]
There's a property on `ProjectReference`s called `ReferenceOutputAssembly` that some of the common build machinery uses to determine whether to add a reference to the winmd/dll/other output from the project.

Right now, C++/WinRT unequivocally includes all winmds produced by static library project references in the list of winmds to pass off to mdmerge. If multiple projects have a dependency on the same static library (for one reason or another), this can result in duplicate type errors from mdmerge.

This should suffice:

```xml
    <CppWinRTStaticProjectReferences Remove="@(CppWinRTStaticProjectReferences)"
                                     Condition="'%(ReferenceOutputAssembly)' != 'true'" />
```

(or: just don't include them to remove them later; this item was written from the perspective of somebody who wanted to remove them after-the-fact.)

**Comment by DHowett-MSFT** [comment]
This may be helped by #491 (/cc @jlaanstra)

**Comment by jlaanstra** [comment]
Yes, it will.

**Comment by jlaanstra** [comment]
This is fixed with #491 and can be closed.
